specs/2026-07-31-sana-theme-optional-design.md

```
@@ -0,0 +1,138 @@
+</CanvasProvider>
+```
+
+**After:**
+```tsx
+<CanvasProvider>
+  <App />
+</CanvasProvider>
+```
+
+The theme prop is now only needed for scoped theming scenarios.
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)